modules/@apostrophecms/attachment/index.js

```diff
@@ -199,6 +199,15 @@ module.exports = {
             return true;
           }
         ]
+      },
+      get: {
+        url(req) {
+          const { attachment, options } = req.query;
+          if (!attachment) {
+            throw self.apos.error('invalid');
+          }
+          return self.url(attachment, options);
```

Comment:
Rather than creating another route to call, you should use the built-in annotation capabilities of `self.all` (apos.attachments.all) to get the URLs as properties the first time. See the documentation of that method, you can pass it an annotate option.

A minor nuisance is that `self.all` expects a parent object with one or more attachments in it as sub-properties. You can just pass it an object with the one attachment as a subproperty to work around that.